qml/MainMenu.qml

```diff
@@ -6,7 +6,6 @@ import QtQuick.Controls.Material 2.2
 import QtQuick.Layouts 1.3
 import Qt.labs.settings 1.0
 
-import Ping1DNamespace 1.0
```

Comment:
This breaks "Emit Ping" button.

Reply:
@patrick I think instead of exporting pingmessage enums to qml, it's better to have a QINVOKABLE emitPing()

Reply:
I added emitPing(), it's less code for the whole project.